README.md

```diff
@@ -115,7 +115,7 @@ Three types of these events exist:
 constructorIO.trackInputFocus(searchTerm: "")
 
 // Track when the user selects an autocomplete suggestion
-constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions")
+constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions", resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```

Comment:
Theres a `group` argument before resultID of type `CIOGroup`, so a full example would like something like 

`constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions", group: CIOGroup(displayName: "groupName1", groupID: "groupID2", path: "path/to/group"), resultID: "179b8a0e-3799-4a31-be87-127b06871de2")`

Reply:
1b84b16